generic3g/transforms/RegridTransform.F90

```diff
@@ -142,13 +143,46 @@ subroutine update(this, importState, exportState, clock, rc)
          call MAPL_FieldBundleGet(fb_in, geom=geom_in, _RC)
          call MAPL_FieldBundleGet(fb_out, geom=geom_out, _RC)
          call this%update_transform(geom_in, geom_out)
-         call this%regrdr%regrid(fb_in, fb_out, _RC)
+         do_transform = check_do_transform(fb_in, fb_out, _RC)
+         if (do_transform) then
+            call this%regrdr%regrid(fb_in, fb_out, _RC)
+         end if
       end if
 
       _RETURN(_SUCCESS)
       _UNUSED_DUMMY(clock)
    end subroutine update
 
+   function check_do_transform(fb_in, fb_out, rc) result(do_transform)
+      logical :: do_transform
+      type(ESMF_FieldBundle), intent(in) :: fb_in
+      type(ESMF_FieldBundle), intent(in) :: fb_out
+      integer, optional, intent(out) :: rc
+
+      logical :: do_trans_in, do_trans_out, present_in, present_out 
+      integer :: status
+
+      call MAPL_FieldBundleIsPresent(fb_in, do_regrid_transform=present_in, _RC)
```

Comment:
OK - I see "why" now, but ...   Can't we simply arrange it so that there is always a value?    Just add a call to FieldBundleSet() in BracketClassAspect::create().